.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -3,37 +3,58 @@
+  - label: ":pipeline::kibana::seedling: Trigger Kibana Serverless Tests for ${ENVIRONMENT}"
+    trigger: appex-qa-kibana-serverless-ftr-tests # https://buildkite.com/elastic/appex-qa-kibana-serverless-ftr-tests
+    soft_fail: true # Remove this before release or when tests stabilize
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":pipeline::fleet::seedling: Trigger Fleet Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Fleet specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - label: ":pipeline::weight_lifter::seedling: Trigger RAC Alert Load tests for ${ENVIRONMENT}"
+    trigger: appex-qa-rac-alert-load # https://buildkite.com/elastic/appex-qa-rac-alert-load
+    soft_fail: true # Remove this before release or when tests stabilize
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+        SERVERLESS_ONLY: true
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":pipeline::lock::seedling: Trigger Security Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Security specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - group: ":female-detective: Security Solution Tests"
+    key: "security"
+    steps:
+      - label: ":pipeline::female-detective::seedling: Trigger Security Solution quality gate script"
+        command: .buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh
 
-  - label: ":pipeline::lock::seedling: Trigger Control Plane Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Control Plane specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+      - label: ":judge::seedling: Trigger Manual Tests Phase"
+        command: "make -C /agent trigger-manual-verification-phase"
+        if: build.branch == "main"
+        env:
+          TEAM_CHANNEL: "#kibana-mission-control"
+        agents:
+          image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.2"
+
+  - label: ":pipeline::ship::seedling: Trigger Fleet serverless smoke tests for ${ENVIRONMENT}"
+    trigger: fleet-smoke-tests # https://buildkite.com/elastic/fleet-smoke-tests
+    soft_fail: true # Remove this before release or when tests stabilize
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":rocket: Run cp e2e tests"
-    trigger: "ess-k8s-qa-e2e-tests-daily"
+  - label: ":pipeline::rocket::seedling: Trigger control-plane e2e tests"
+    trigger: "ess-k8s-qa-e2e-tests-daily" # https://buildkite.com/elastic/ess-k8s-qa-e2e-tests-daily
+    soft_fail: true # Remove this before release or when tests stabilize
```

Comment:
I wasn't aware that this pipeline was flaky and hence required us to use `soft_fail: true` to make it non-blocking?